v3/lints/mozilla/lint_mp_ecdsa_signature_encoding_correct.go

```diff
@@ -94,6 +97,8 @@ func (l *ecdsaSignatureAidEncoding) Execute(c *x509.Certificate) *lint.LintResul
 	const maxP256SigByteLen = 72
 	// len <= 2+2+2+49+49 (= 104)
 	const maxP384SigByteLen = 104
+	// len <= 2+2+2+67+67 (= 140)
```

Comment:
It took me a bit to confirm this, which means it took you longer than that to write it :laughing: thank you!